android-test/src/androidTest/java/okhttp/android/test/OkHttpTest.kt

```diff
@@ -67,13 +70,24 @@ class OkHttpTest {
 
   @Before
   fun createClient() {
-    client = OkHttpClient.Builder()
-        .build()
+    client = OkHttpClient.Builder().build()
   }
 
   @After
   fun cleanup() {
-    client.dispatcher.executorService.shutdownNow()
+    if (this::client.isInitialized) {
```

Comment:
can you just assume it’s initialized?

Reply:
I can now, it's mainly if the test setup is broken this obscures the root exception.